### ISSUE TITLE
What questions users are asking re:status of Searchworks?

### ISSUE THREAD

**Issue by cmh2166** [sul-dlss/SearchWorks#1893]
What are the questions users are asking about for the status of Searchworks & its upstream indexing pipelines? 

Perhaps start by enumerating & reviewing where this has been asked in the past?

**Comment by cmh2166** [comment]
Opening a ticket here to give a space for folks to coordinate notes, questions, answers to this.

**Comment by ahafele** [comment]
Some info from the SW Feedback JIRA dive. 

SW slowness reports started in 2009. 
Around 70 tickets (but includes Requests related reports)

Types of Slowness
- Large records
- Availability
- Large facets (old reports)
- Search Results (most common)

Findings
- People think we already know. 
- Reports are happening days, weeks, sometimes a month after slowness has started. Presumably related to first point. 
- They want to know why it is slow. 

Inference from tickets/emails
- Staff does not feel heard and needs a way to communicate "critical" sw issues directly.
- Updates on progress need to be communicated clearly
- need monitoring so we can confirm reports and all be on same page (I know we know this one already :).

**Comment by shelleydoljack** [comment]
When something updates in Symphony, when does it make it to SearchWorks? We always say around 2 hours you'll see the update but I think users aren't quite satisfied with that.

**Comment by jvine** [comment]
Users want a web page they can refer to:
- they aren't all on slack
- they find the current twitter feed (https://twitter.com/SULSystemStatus) hard to follow for problems and resolutions, intermingled with database/SFX status etc.
- the problem might have been ongoing for a while, or they may be checking on it some time after they experienced it; they want to easily see if it was noticed, when it started, progress, etc.

Page should be linked from the affected page (e.g. search results; "we're sorry, something went wrong", etc.), as well as from the Feedback form.

Public-facing staff want to feel as informed as possible when patrons are asking them about SW issues. They would appreciate:
- more information than the general public may need (e.g. cause, degree) 
- email notification when a service degradation is noticed, with some idea of time to resolution. 

Users want to know:
**1. do we know about it, and are we working on it?**
**2. when will it be over?**
3. when it started 
4. how bad is it? (e.g. how does this slowdown compare to normal performance?)
5. what caused it?

In addition to Alissa's point that people report when the problem has been ongoing for a while, we also get reports of issues after they have been resolved (magically or by intervention). From a support/trust perspective, it would be helpful to be able to show: yes, this blip happened at 10:30, this is why, it's normal now. That implies: a) some measure of what is normal, b) a timeline of events, maybe showing number of users or other factors that might contribute to performance issues. 

Other information of interest:
- number & schedule of records added (see Shelley's comment)
- average response time
- size of the database (?)

One librarian suggested we consider integrating SW alerts with the UIT Service Now alerts. Which is maybe not a bad idea for substantial or planned outages?

**Comment by aeschylus** [comment]
Can someone post a link here to some of the related JIRA tickets (or slack it to me)? It would be useful for me to see how the users word specific reports.

**Comment by ahafele** [comment]
@aeschylus [JIRA](https://jirasul.stanford.edu/jira/issues/?jql=project%20in%20(SW%2C%20VUF)%20AND%20text%20~%20%22slow%20NOT%20requests%22%20ORDER%20BY%20created%20DESC)
I tried to weed out tickets specific to the requests app.

**Comment by cmh2166** [comment]
Anything else to add to this? This seems like a good gathering of info, and we can close this and move to focusing on synthesis of info.

**Comment by jvine** [comment]
See Slack : https://stanfordlib.slack.com/archives/C2ZDF3TS8/p1532372837000245

**Comment by cmh2166** [comment]
Can continue to use for any ongoing aggregation of info, but closing for purposes of SW Indexing & Performance work cycle.

**Comment by aeschylus** [comment]
https://docs.google.com/document/d/1jGhlX6hBtxuTw1G1NsAAYO5U9sYXddTctNt3OiCT_v0/edit#heading=h.ga3roeaqymz3
